### ISSUE TITLE
Repeat crash CameraManager.swift line 430 in release

### ISSUE THREAD

**Issue by brod-ie** [imaginary-cloud/CameraManager#138]
We're seeing this issue repeatedly in product using CamerageManager 4.0.1:
```
Crashed: CameraSessionQueue
0  CameraManager                  0x86c860 specialized CameraManager.(capturePictureDataWithCompletion((Data?, NSError?) -> ()) -> ()).(closure #1) (CameraManager.swift:430)
1  CameraManager                  0x86c74c specialized CameraManager.(capturePictureDataWithCompletion((Data?, NSError?) -> ()) -> ()).(closure #1) (CameraManager.swift:430)
2  CameraManager                  0x872544 partial apply for CameraManager.(capturePictureDataWithCompletion((Data?, NSError?) -> ()) -> ()).(closure #1) + 9569604
3  CameraManager                  0x860b30 thunk (CameraManager.swift)
4  libdispatch.dylib              0x1bb99797 _dispatch_call_block_and_release + 10
5  libdispatch.dylib              0x1bba659d _dispatch_queue_serial_drain + 854
6  libdispatch.dylib              0x1bb9cb71 _dispatch_queue_invoke + 886
7  libdispatch.dylib              0x1bba6a1f _dispatch_queue_override_invoke + 282
8  libdispatch.dylib              0x1bba81b5 _dispatch_root_queue_drain + 326
9  libdispatch.dylib              0x1bba800f _dispatch_worker_thread3 + 106
10 libsystem_pthread.dylib        0x1bd4e87d _pthread_wqthread + 1040
11 libsystem_pthread.dylib        0x1bd4e45c start_wqthread + 8
```

Any ideas?

**Comment by Splash04** [comment]
Looks like that case should be implemented:
https://stackoverflow.com/a/30353873

**Comment by torrao** [comment]
@Splash04 thanks for the suggestion.

@brod-ie this should be fixed in v4.0.2
